### PR TITLE
Mobile Styling for Report page

### PR DESCRIPTION
**Pull request by IsmatMahmood1**
https://trello.com/c/fHhJfvIq/35-mobile-styling-for-report-page
image
image

### CODE REVIEW

File: WhaleSpotting/react-app/src/styles/ReportSighting.scss [hunk 1]
```diff
@@ -80,4 +81,29 @@
...
+        }
+        .card-component{
+            padding-right: 7px;
+        }
+        .submit-button{
+            margin: 10px 0 25px 0;
+        }
+
+    }
+    @media (max-width: $minMobileScreenSize){
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/ReportSighting.scss:102]: we should also add changing the margin bottom to mobile footerHeight in this

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: WhaleSpotting/react-app/src/styles/ReportSighting.scss (hunk 1 above)
Comment (IsmatMahmood1) [reply WhaleSpotting/react-app/src/styles/ReportSighting.scss:102]: Added as requested

Comment (oskarwilliams) [reply WhaleSpotting/react-app/src/styles/ReportSighting.scss:102]: The change should edit the padding-bottom of the root report-sighting class, not the container. Also it should happen in the second media query not the first

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: WhaleSpotting/react-app/src/styles/ReportSighting.scss [hunk 2]
```diff
@@ -80,4 +81,32 @@
...
+        .title{
+            font-size: $smallMobileTitleFontSize;
+        }
+        .report-sighting{
+            padding-bottom: $mobileFooterHeight;
+        }
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/ReportSighting.scss]:
```suggestion
        padding-bottom: $mobileFooterHeight;

        .title{
            font-size: $smallMobileTitleFontSize;
        }
```

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**